### PR TITLE
Make drag and drop autoscroll accelerate more near edges.

### PR DESCRIPTION
**Pull request by Kiisu-Master**
In case someone is confused, here is a graph.
m is the `scroll_area`.
https://www.desmos.com/calculator/msp5v3av4e

### CODE REVIEW

**Comment by Kiisu-Master** [comment]
Uhhh something is wrong with the safe margin.

**Comment by MewPurPur** [comment]
So this doesn't work correctly in your PR yet?

**Comment by Kiisu-Master** [comment]
It works fine. I just noticed that for some weird reason the zone in which autoscroll starts isn't of the correct size ( 1 / 6 ). It's a bit bigger, but also not twice as big. Idk why it is like that...

**Comment by Kiisu-Master** [comment]
Also i believe this can still be improved. Right now its linearly increasing and max speed depends on the size of the scroll container.

**Comment by Kiisu-Master** [comment]
give me a moment i have an idea

File: src/ui_parts/tag_container.gd [hunk 1]
```diff
@@ -1,6 +1,8 @@
 extends PanelContainer
 
-const safe_margin = 1 / 6.0
+# Autoscroll area on drag and drop. As a factor from edge to center.
+const safe_margin := 1.0 / 2.0
```
Comment (Kiisu-Master) [inline src/ui_parts/tag_container.gd:4]: Maybe i should rename the variable... its not exactly the safe margin.
Its the scroll area.

Comment (MewPurPur) [reply src/ui_parts/tag_container.gd:4]: Sure.

Are you sure about 1/2? Seems quite extreme.

Comment (Kiisu-Master) [reply src/ui_parts/tag_container.gd:4]: 1/2 is actually 1/4 of the scroll container height.
Also keep in mind scrolling is slower near the center.

Comment (MewPurPur) [reply src/ui_parts/tag_container.gd]: I know, and I know, but it seems extreme nonetheless. You normally only have scrolling towards the very edges, the central part is for dropping things without repercussions.

**Comment by MewPurPur** [comment]
Could you show a clip of how it looks?

File: src/ui_parts/tag_container.gd (hunk 1 above)
Comment (MewPurPur) [reply src/ui_parts/tag_container.gd]: Tweaked it from 1/2 to 1/3, it seemed_ best to me that way. With 1/2, I had situations where I grab a tag and it immediately starts scrolling.

**Comment by MewPurPur** [comment]
Tested it and I really like how it works!

**Pull request merged**